airtable/generate_chapter_data.py

```diff
@@ -17,6 +19,15 @@ def get_lat_long(s):
     else:  # fuuuuck it's an address, ask google about how to make that a lat long
         payload = {"address": s}
         r = requests.get("http://maps.googleapis.com/maps/api/geocode/json", params=payload).json()
+
+        tries = 1
+        while r["status"] == "UNKNOWN_ERROR" and tries < GEOCODING_RETRIES:
```

Comment:
That's a good point, but I didn't want to retry on [any kind of error](https://developers.google.com/maps/documentation/geocoding/intro#StatusCodes). If any other type of error happens we can revisit this.